src/test/operators/jit_operator_wrapper_test.cpp

```diff
@@ -244,6 +248,61 @@ TEST_F(JitOperatorWrapperTest, FilterTableWithLiteralAndParameter) {
                                 FloatComparisonMode::AbsoluteDifference));
 }
 
+TEST_F(JitOperatorWrapperTest, FilterTableOnValueIDs) {
```

Comment:
There is a BETWEEN test in `JitAwareLQPTranslatorTest::CreatesValueIDExpressions` as the `JitAwareLQPTranslator` translates a BETWEEN to two JitExpressions with <= and >=. Since a BETWEEN cannot be explicitly expressed with a JitExpression I do not see the necessity for a further BETWEEN test.